bpfd-operator/config/bpfd-deployment/config.yaml

```diff
@@ -7,7 +7,10 @@ data:
   ## Can be configured at runtime
   bpfd.agent.image: quay.io/bpfd/bpfd-agent:latest
   bpfd.image: quay.io/bpfd/bpfd:latest
-  bpfd.log.level: "info"
+  ## Can be set to "info", "debug", or "trace"
```

Comment:
Update the docs.
docs/developer-guide/logging.md

Reply:
Done!